python/llm/dev/test/run-example-ipynb-tests.sh

```diff
@@ -0,0 +1,22 @@
+if [ -z "$THREAD_NUM" ]; then
+  THREAD_NUM=2
+fi
+export OMP_NUM_THREADS=$THREAD_NUM
+export LLM_INFERENCE_TEST_DIR=./llm/test/langchain_gpu
+
+echo ">>> Testing LangChain upstream unit test"
+mkdir ${LLM_INFERENCE_TEST_DIR}/langchain_upstream_ut
+wget https://raw.githubusercontent.com/langchain-ai/langchain/master/libs/community/tests/integration_tests/llms/test_bigdl.py -P ${LLM_INFERENCE_TEST_DIR}/langchain_upstream_ut
+sed -i "s,model_id=\"[^\"]*\",model_id=\"$LLAMA2_7B_ORIGIN_PATH\",g" ${LLM_INFERENCE_TEST_DIR}/langchain_upstream_ut/test_bigdl.py
+python -m pytest -s ${LLM_INFERENCE_TEST_DIR}/langchain_upstream_ut
+rm -rf ${LLM_INFERENCE_TEST_DIR}/langchain_upstream_ut
+
+echo ">>> Testing LangChain upstream ipynb"
+mkdir ${LLM_INFERENCE_TEST_DIR}/langchain_upstream_ipynb
```

Comment:
use one folder is enough? langchain_upstream?

Reply:
Sure, I'll use only one folder.